requirements.txt

```diff
@@ -1,29 +1,8 @@
-asgiref==3.3.1
-astropy>=4.2
-astroquery==0.4.3
-boto3==1.17.59
-certifi==2020.12.5
-click==7.1.2
-cycler==0.10.0
-Django==3.1.7
-django-tables2==2.4.0
-Flask==1.1.2
-greenlet==1.0.0
-itsdangerous==1.1.0
-Jinja2==2.11.3
-kiwisolver==1.3.1
-MarkupSafe==1.1.1
-matplotlib==3.3.4
-moto==2.0.5
-numpy==1.20.1
-photutils==1.1.0
-Pillow==8.1.2
-pyerfa==1.7.2
-pyparsing==2.4.7
-python-dateutil==2.8.1
-pytz==2021.1
-scipy==1.7.1
-six==1.15.0
-SQLAlchemy==1.4.1
-sqlparse==0.4.1
-Werkzeug==1.0.1
+astropy==5.0.1
+astroquery==0.4.5
+boto3==1.21.2
+Django==4.0.2
+matplotlib==3.5.1
+moto==3.0.3
+numpy==1.22.2
+Pillow==9.0.1
```

Comment:
```suggestion
Pillow==9.0.1
pyqt5>=5.0
```

Reply:
It's not quite ready yet, perhaps I should have draft PR it but then the GA tests wouldn't have run and I was testing GA, the python version was supposed to be bumped to 3.10 in GA once I was done and history cleaned up. This was just getting rid of errors and warnings I saw when locally deploying against Python 3.10. 

Sorry for the confusion.